packages/flutter/test/rendering/proxy_box_test.dart

```diff
@@ -261,6 +264,9 @@ class _FakeTicker implements Ticker {
   @override
   String get debugLabel => null;
 
+  @override
+  TimeDilationBehavior timeDilationBehavior;
```

Comment:
Perhaps it would be no less of a fake ticker, but why not return ...normal here?

Reply:
Done